### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by HMPoy**
The code seems to work like this for the value checked:[ ] thx.

## Description of Pull Request:

Please replace this line with a detailed description of your pull request. If no description your PR
may be rejected.

## PR checklist:

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Enhancement to an existing feature
- [ ] ARIA accessibility
- [x] Documentation update
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [x] Yes
- [ ] No

If yes, please describe the impact: this allows the checked array to be retrieved when the form is sent.

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch, **not** the `master` branch
- [x] When resolving a specific issue, it's referenced in the PR's title (i.e.
      `fixes #xxxx[,#xxxx]`, where "xxxx" is the issue number)
- [x] The PR should address only one issue or feature. If adding multiple features or fixing a bug
      and adding a new feature, break them into separate PRs if at all possible.
- [x] PR titles should following the
      [**Conventional Commits**](https://www.conventionalcommits.org/) naming convention (i.e.
      "fix(alert): not alerting during SSR render", "docs(badge): Updated pill examples, fix typos",
      "chore: fix typo in docs", etc). **This is very important, as the `CHANGELOG` is generated
      from these messages.**

**If new features/enhancement/fixes are added or changed:**

- [x] Includes documentation updates (including updating the component's `package.json` for slot and
      event changes)
- [ ] New/updated tests are included and passing (if required)
- [ ] Existing test suites are passing
- [x] The changes have not impacted the functionality of other components or directives
- [ ] ARIA Accessibility has been taken into consideration (does it affect screen reader users or
      keyboard only users? clickable items should be in the tab index, etc)

**If adding a new feature, or changing the functionality of an existing feature, the PR's
description above includes:**

- [x] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a
      suggestion issue first and wait for approval before working on it)

### CODE REVIEW

File: src/components/form/README.md [hunk 1]
```diff
@@ -40,9 +40,9 @@
       </b-form-group>
 
       <b-form-group id="input-group-4">
-        <b-form-checkbox-group v-model="form.checked" id="checkboxes-4">
```
Comment (tmorehouse) [inline src/components/form/README.md]: This is the intended place for v-model in this example.

**Comment by tmorehouse** [comment]
The position of the `v-model` on the check box group in this example is where it should be.

**Comment by HMPoy** [comment]
I have not taken the time to compare with a nuxt installation. For my part install via npm. Does not work on the checkbox_group tag.

**Comment by tmorehouse** [comment]
Works in docs site https://bootstrap-vue.js.org/docs/components/form

If you are running a local copy of bootstrap-vue docs, make sure you are running the latest version.

**Comment by HMPoy** [comment]
OK. 2 version vuejs 2.6.10 loaded. In app.js and template. Work thx.

**Pull request closed without merging**